driver/bpf/fillers.h

```diff
@@ -7111,4 +7111,74 @@ FILLER(sched_prog_fork_3, false)
 }
 #endif
 
+FILLER(sys_prctl_x, true)
+{
+	unsigned long option;
+	unsigned long arg2;
+	unsigned long arg2_int;
+	int res;
+	long retval;
+
+	retval = bpf_syscall_get_retval(data->ctx);
+	res = bpf_val_to_ring(data, retval);
+	if (res != PPM_SUCCESS)
+		return res;
+
+	/*
+	 * option
+	 */
+	option = prctl_options_to_scap(bpf_syscall_get_argument(data, 0));
+	res = bpf_val_to_ring(data, option);
+	if (res != PPM_SUCCESS)
+		return res;
+
+	arg2 = bpf_syscall_get_argument(data, 1);
+
+	switch(option){
+		case PPM_PR_SET_NAME:
+			/*
+			 * arg2_str
+			 */
+			res = bpf_val_to_ring(data, arg2);
+			if (res != PPM_SUCCESS)
+				return res;
+			/*
+			 * arg2_int
+			 */
+			res = bpf_val_to_ring(data, 0);
+			if (res != PPM_SUCCESS)
+				return res;
+			break;
+		case PPM_PR_GET_CHILD_SUBREAPER:
+			/*
+			 * arg2_str
+			 */
+			res = bpf_val_to_ring(data, 0);
+			if (res != PPM_SUCCESS)
+				return res;
+			/*
+			 * arg2_int
+			 */
+			bpf_probe_read_user(&arg2_int,sizeof(arg2_int),(void*)arg2);
+			res = bpf_val_to_ring(data, (int)arg2_int);
+			break;
+		default:
```

Comment:
In the default case, do we want to push both params as empty?

Reply:
Maybe we could be a little conservative: I think we can keep arg2_int with the actual value (I think that it's the most used argument) and keep arg2_str empty. WDYT? @FedeDP